express-get-json/index.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express');
+const app = express();
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+}
+
+const jsonArr = [];
```

Comment:
This and the loop below must be in the .get method call code block.

Reply:
![image](https://user-images.githubusercontent.com/73314663/103833635-bf92bc00-5081-11eb-86b5-85a9f7decba3.png)

Done! May I please know why? I couldn't tell the difference between declaring and running the for...in loop outside and within the app.METHOD.